src/server/api/index.js

```diff
@@ -573,6 +573,29 @@ function createAPI(app, mountPath, middlewareOpts) {
     });
 
 
+    router.put('/projects/:ownerId/:projectName', function (req, res, next) {
+        var userId = getUserId(req),
+            command = req.body;
```

Comment:
Add a comment here what kind of fields should be/can be passed by the body.

Like type is mandatory and seedName?
Add two examples in the comment.